E1_TFHub_Sample_Deploy/export.py

```diff
@@ -1,10 +1,17 @@
+import sys
 import tensorflow as tf
 import tensorflow_datasets as tfds
 import tensorflow_hub as hub
+from absl import app, flags
 """ Create a Sample TF-Hub Module using SavedModel v2.0
 The module has as a single signature which loads MNIST Dataset from TFDS and train a simple Neural Network for classifying the digits. The model is built and trained using Tewnsorlfow
 """
-
+FLAGS = flags.FLAGS
```

Comment:
Do not define these directly as command-line arguments. Create functions that have these arguments, which can then be called programmatically in unit tests. Then, in the `if__name__ ==...` section, parse command line flags and call the functions via main, as needed.

This enables this code to be used as a library as well as a script. Otherwise, there is no way to run it except as a command line script. Unit tests require functions to be called, they don't invoke a script.

For example, see [here](https://github.com/tensorflow/hub/blob/master/examples/text_embeddings/export.py#L180). `export_path` is an argument to a function, and then [optionally taken as a parameter](https://github.com/tensorflow/hub/blob/master/examples/text_embeddings/export.py#L262) _if_ the code is used as a script.

Reply:
Okay  :)

Reply:
Done :)